ci-operator/config/openshift/bushslicer/openshift-bushslicer-master.yaml

```diff
@@ -0,0 +1,51 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: plugin-base
```

Comment:
Do they have this image? It was one from `azure` namespace we produced. they will need to either use some default one or create their own.

Reply:
They do not.  They use 
```
FROM openshift/jenkins-slave-base-centos7
```
https://github.com/openshift/BushSlicer/blob/master/tools/jenkins/slaves/Dockerfile.centos#L3

Reply:
Need to check if this is available on published images for them to use...

Reply:
@kargakis, yes, I copied ours as a template.

Reply:
@mjudeikis, looks like it is here: https://hub.docker.com/r/openshift/jenkins-slave-base-centos7/

Reply:
`jenkins-slave-base-centos7` is an official OpenShift image. I hope it is still being maintained upstream.